src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -9,6 +9,8 @@ import { doChallenge } from '../../utils/ChallengeViewUtil';
 import OktaVerifyAuthenticatorHeader from '../../components/OktaVerifyAuthenticatorHeader';
 import { getSignOutLink } from '../../utils/LinksUtil';
 
+const SIGNED_NONCE_ERROR_KEY = 'auth.factor.signedNonce.error';
```

Comment:
nit: Doesn't need to be a top-level constant. Let's move this within the scope of the function using it.

Reply:
updated